scripts/darwin.sh

```diff
@@ -1,35 +1,38 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
I did re-run shellcheck after changing to bash (sorry for the confusion of putting the initial warnings there. I'll update). shellcheck basically stopped request POSIX compliance stuff but didn't add anything new.